code/modules/robotics/bot/guardbot.dm

```diff
@@ -2759,6 +2764,11 @@ TYPEINFO(/obj/item/device/guardbot_module)
 							return
 
 						if(BOUNDS_DIST(master, hug_target) == 0)
+							if(hug_target.traitHolder?.hasTrait("wasitsomethingisaid"))
+								master.bot_attack(hug_target, TRUE) //betrayal!!
+								master.speak(pick("As if!", "You know what you did.", "Level [rand(32,80)] dork alert!"))
+								drop_hug_target()
+								return
```

Comment:
normal hug behaviour sets `master.moving = 0` before returning. Intentional that this doesn't?

Reply:
was not intended fixed alongside other suggested change, thanks!